Source/ASCellNode.mm

```diff
@@ -315,13 +315,13 @@ - (void)handleVisibilityChange:(BOOL)isVisible
     if (ip != nil) {
       [result addObject:@{ @"indexPath" : ip }];
     }
-    [result addObject:@{ @"collectionNode" : ASObjectDescriptionMakeTiny(owningNode) }];
```

Comment:
Longer term, I wonder if a separate file like ASLog+DebugDescriptions.m would be a good place to locate category implementations for all the node classes. One-stop shop for logging contents with helper macros and such. Locating it in the class like this might be better for other reasons, though.